src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -28,28 +28,28 @@ const endIndeterminate = 80;
 /* Maximum angle for the arc. The angle can't be exactly 360, because the arc becomes hidden. */
 const MAX_ANGLE = 359.99 / 100;
 
-export type ProgressCircleMode = 'determinate' | 'indeterminate';
+export type ProgressSpinnerMode = 'determinate' | 'indeterminate';
 
 type EasingFn = (currentTime: number, startValue: number,
                  changeInValue: number, duration: number) => number
 
 
 /**
- * <md-progress-circle> component.
+ * <md-progress-spinner> component.
  */
 @Component({
   moduleId: module.id,
-  selector: 'md-progress-circle, mat-progress-circle',
+  selector: 'md-progress-spinner, mat-progress-spinner',
   host: {
     'role': 'progressbar',
     '[attr.aria-valuemin]': '_ariaValueMin',
     '[attr.aria-valuemax]': '_ariaValueMax',
   },
-  templateUrl: 'progress-circle.html',
-  styleUrls: ['progress-circle.css'],
+  templateUrl: 'progress-spinner.html',
+  styleUrls: ['progress-spinner.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MdProgressCircle implements OnDestroy {
+export class MdProgressSpinner implements OnDestroy {
```

Comment:
You can add at the bottom of this file
```ts
/** @deprecated */
export {MdProgressSpinner as MdProgressCircle};
/** @deprecated */
export {MdProgressSpinnerModule as MdProgressCircleModule};
// same for other symbols
```

this will let people continue using the old symbols and let us migrate them incrementally.